webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -749,6 +749,8 @@ Kommunicate.richMsgEventHandler = {
         console.log('passenger detail submitted');
     },
     processQuickReplies: function (e) {
+        eventMapping.richMessageButtonEvent.data.eventLabel(e.target.innerText);
+        kmWidgetEvents.eventTracking(eventMapping.richMessageButtonEvent);
```

Comment:
can these 2 be combined to a single line, eg:
kmWidgetEvents.eventTracking(eventMapping.richMessageButtonEvent, e.target.innerText);
then have the eventTracking function handle it internally